packages/core/admin/admin/src/components/LeftMenu.tsx

```diff
@@ -135,6 +136,20 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }: LeftMenuProps) =
       <Divider />
 
       <NavSections>
+        {condensed && (
+          <NewNavLink.Link to="/">
+            <NewNavLink.Tooltip
+              label={formatMessage({ id: 'global.home', defaultMessage: 'Home' })}
+            >
+              <>
+                <NewNavLink.Icon>
+                  <Icon as={House} color="neutral500" />
+                </NewNavLink.Icon>
+                <NewNavLink.Badge label="ee only">5</NewNavLink.Badge>
```

Comment:
is that label correct? 👀

Reply:
you are right, I need to remove the Badge 😅  it was just for testing